src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -400,23 +414,25 @@ private static BattleshipModelUpdated computerFireEasy(BattleshipModelUpdated mo
         }
 
         j = model.getPlayerHits().size();
-        Point Clipper = new Point((PArray[3].getStart().getAcross() + PArray[3].getEnd().getAcross())/2,(PArray[3].getStart().getDown() + PArray[3].getEnd().getDown())/2);
-
 
         // The following branch tree checks if a point fired at
         // BY A PLAYER has hit a COMPUTER ship and adds the point to the array of hits if so
         for (int i = 0; i < 5; i++) {
             if (Hit(PArray[i].getStart(), PArray[i].getEnd(), FireSpotComputer)) {
-                /*if(i == 2){
-                    model.addPointtoArray(PArray[i].getStart(), model.getPlayerHits());
-                    model.addPointtoArray(PArray[i].getEnd(), model.getPlayerHits());
-                    model.addPointtoArray(Clipper, model.getPlayerHits());
 
-                }else {*/
                 model.addPointtoArray(FireSpotComputer, model.getPlayerHits());
-                //}
-
+                //System.out.println("Computer");
+                //System.out.println(model.getShipByID(PArray[i].getName()).getHealth());
                 model.getShipByID(PArray[i].getName()).setHealth(PArray[i].getHealth() - 1);
+                /*System.out.println(model.getShipByID(PArray[i].getName()).getHealth());
```

Comment:
pseudo code for sinking and will either add it or delete it on the final. This is what your issue is about so added trace statements so that you can follow what my pseudo code does.